schedule/virt_autotest/alp-virt-core-tests.yaml

```diff
@@ -0,0 +1,9 @@
+name:           alp-virt-core-tests
+description:    >
+    Maintainer: xlai@suse.com, qe-virt@suse.de
+    ALP virtualization guest installation and virt feature tests schedule
+schedule:
+    - virt_autotest/login_console
+    - virt_autotest/setup_kvm_container
```

Comment:
Will this test module setup_kvm_container work on SLES/SLE-Micro/Tumbleweed/ALP host ?

Reply:
Based on project test need, only ALP requies it.